catalog/log-export/org/bigquery-export/setters.yaml

```diff
@@ -3,14 +3,14 @@ kind: ConfigMap
 metadata:
   name: setters
 data:
+  # This is required
+  project-id: my-project-id
+
```

Comment:
Is the extra line to prevent kpt reordering bug? Noticed [this](https://github.com/GoogleCloudPlatform/blueprints/blob/466f2cea889a115d7c2c11832f47c37097ae9052/catalog/networking/network/setters.yaml) doesn't have it.

Reply:
It was an attempt to, but didn't work. I have removed the newline.